packages/core/exports/react.ts

```diff
@@ -20,14 +20,23 @@ export function useAppKitNetworkCore(): Pick<
 }
 
 export function useAppKitAccount(): UseAppKitAccountReturn {
-  const { status } = useSnapshot(AccountController.state)
+  const { status, email, username, preferredAccountType, smartAccountDeployed } = useSnapshot(
+    AccountController.state
+  )
+
   const { activeCaipAddress } = useSnapshot(ChainController.state)
 
   return {
     caipAddress: activeCaipAddress,
     address: CoreHelperUtil.getPlainAddress(activeCaipAddress),
     isConnected: Boolean(activeCaipAddress),
-    status
+    status,
+    user: {
```

Comment:
shouldn't this be part of wallet info instead? it's not really user data but rather wallet data which has user info from auth/email/SA

Reply:
nvm